packages/react-utilities/src/utils/getNativeElementProps.ts

```diff
@@ -56,14 +56,13 @@ const nativeElementMap: Record<string, Record<string, number>> = {
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function getNativeElementProps<TAttributes extends React.HTMLAttributes<any>>(
-  tagName: string,
+  tagName: keyof JSX.IntrinsicElements,
   props: {},
   excludedPropNames?: string[],
```

Comment:
This could possibly result in too big of a change, but it would be nice to use an args object so the arguments are named, like `getPartitionedNativeProps` has.